build.gradle

```diff
@@ -255,6 +255,13 @@ project(':cruise-control') {
       exclude group: 'org.slf4j', module: 'slf4j-log4j12'
       exclude group: 'log4j', module: 'log4j'
     }
+
+    constraints {
+      implementation('com.fasterxml.jackson.core:jackson-databind:2.12.6.1') {
+        because 'Prior version is vulnerable. Can be unpinned after Apache Kafka 3.1.1 is released and pulled in.'
+      }
+    }
+
```

Comment:
Changed to use `api` over `implementation`.  However, after checking I think we should put it in both `cruise-control` and `cruise-control-metrics-reporter` project, because `cruise-control` also transitively pulls `jackson-databind` via `swagger` package.  While with the constraint only in `cruise-control-metrics-reporter` would actually bump it up without putting it in `cruise-control`, I think we should explicitly list the dependency constraint in both module